tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot

```diff
@@ -142,6 +142,17 @@ Spawn Notebook With Arguments
       END
       Wait Until Element is Visible  id:progress-bar
       Run Keyword And Continue On Failure  Wait Until Page Does Not Contain Element  id:progress-bar  ${spawner_timeout}
+      Wait for JupyterLab Splash Screen  timeout=30
+      Maybe Close Popup
+      ${is_launcher_selected} =  Run Keyword And Return Status  JupyterLab Launcher Tab Is Selected
+      Run Keyword If  not ${is_launcher_selected}  Open JupyterLab Launcher
+      Open With JupyterLab Menu  File  New  Notebook
+      Sleep  1
+      Maybe Close Popup
+      Close Other JupyterLab Tabs
+      Maybe Close Popup
+      Sleep  1
+      Run Cell And Check Output  import os; print(os.environ["JUPYTER_IMAGE"].split("/")[-1].split(":")[0])  ${image}
```

Comment:
As @bdattoma I also think the cell used to check the image version should be removed after running it.  Also, I think it should be implemented in a keyword, similar to `Python Version Check`
```
Jupyter Image Version Check  (or even better Verify Jupyter Image Version)
  [Arguments]  ${expected_image}
  Create Temp  Notebook
  Run Cell And Check Output  import os; print(os.environ["JUPYTER_IMAGE"].split("/")[-1].split(":")[0])   ${expected_image}
  Close Temp  Notebook
```

Reply:
I agree it can be moved to a keyword, but not sure if we want to close the document. Right now the TCs are set up expecting an open nb, and if we change this behaviour it would mean duplicating these lines in all TCs:

```
Open With JupyterLab Menu  File  New  Notebook
Sleep  1
Maybe Close Popup
```

Reply:
Maybe you can delete the cell with `Open With JupyterLab Menu    Edit    Delete Cells`   or  using `Execute JupyterLab Command`. If not, closing the notebook, deleting it, and creating a new one looks good to me.

Another option is to implement `Jupyter Image Version Check` using `Run   oc exec` like in `Delete Folder In User Notebook` to get the environment variable value.